deepchem/rl/a3c.py

```diff
@@ -263,7 +263,7 @@ def create_rollout(self):
     for i in range(self.a3c.max_rollout_length):
       if self.env.terminated:
         break
-      state = self.env.state
+      state = copy.deepcopy(self.env.state)
```

Comment:
We should definitely document it for people writing environments.  It's not just this one line—it would create opportunities for bugs all over the place.  Any piece of code that ever stores a state would start having to worry that state might magically change behind its back.